README.md

```diff
@@ -1,4 +1,71 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # ruby_tic_tac_toe
 
-A tic tac toe game made with Ruby.
-This program uses the concepts of Ruby Object Oriented Programming.
+> In this project, we're required to create a Tic Tac Toe Game using Ruby with OOP concepts.
+
+This project is split into 4 milestones, which are:
+
+- Setting up the project
+- Designing the user interface
+- Creating the game logic
+- Writing the game instructions
+
+The main goal of this project is to ensure the mastery of the methods ruby provides and to learn how to replicate them using basic coding knowledge.
+
+Below is a basic demonstration of how the tic tac toe game works
+
+![screenshot](./images/tic.gif)
+
+## Steps to play the game : 
+
+- Player 1 is asked to choose between 'X' and 'O'. Player 2 is assigned the key not chosen by Player 1.
+
+- The game starts with an empty board.
+
+- Players are prompt to enter the position for their respective key.
+
+- The game continues until we have a winning combination or the grid is full and it is a draw.
+
+- Winning combinations are full row, full column or full diagonal.
+
+## How to install
+
+- Download or clone the repository's files github repo
+- Have Ruby 2.7 installed
+- Once repo is downloaded to local directory, in you console run cmd: main
+- Enjoy the game!
```

Comment:
## Elaborate
- Elaborate on how to start a game
  - [ ] Adding a repository link
  - [ ] Rename the title to "How to start game"
  - [ ] Explain how to run the main file